### ISSUE TITLE
built-in boundaries of Gaussian models

### ISSUE THREAD

**Issue by luisfabib** [luisfabib/DeerAnalysis2#31]
As discussed in #29 the upper bounds of the built-in Gaussian parametric models is set to 20nm. This appears to be too large, as in some cases, the ``fitparamodel`` solvers will try to remove a Gaussian from the model by setting it to a large distance (>10nm), instead of by reducing its amplitude. 

By comparison with the boundaries used for other software (notation ``[lower upper]``): 

| Parameter |   DeerLab     | DeerAnalysis2018 | DeerAnalysis2019 |   DD  |
|:-------------:|:---------------:|:----------------:|:----------------:|:------:|
 | ``rmean`` |   ``[1.5  20]`` | ``[1.5 20]``  |  ``[1.5 10]`` |   ``[1.5 5]`` |
 | ``fwhm`` |   ``[0.2 5]`` | ``[0.12 11.8]``|  ``[0.12 11.8]``  |   ``[0.12 5.89]`` |
 | ``amp`` |   ``[0 1]`` | ``[0 1]``|  ``[0 1]``  |   ``[0 1]`` |

One can see that the DeerLab Gaussian function were built based on the DeerAnalysis2018 bounds. It seems these were fixed in the newer DeerAnalysis2019 release. DD uses a maybe too small lower boundary. 

- DeerLab should adopt the same upper boundary as DeerAnalysis2019.

For the width of the Gaussian, DeerLab has more constrained boundaries than DD & DeerAnalysis. 

- These boundaries could maybe be relaxed in DeerLab.

**Comment by luisfabib** [comment]
To add some backbone to the decision, I benchmarked the performance of the ``fitmultigauss`` function with different upper bounds for the mean distances of the Gaussian in the range ``[4 20]``. For the benchmarked I analyzed the clustered subset of the  T4 lysozyme library. 

![untitled](https://user-images.githubusercontent.com/48292540/75354995-2403bd00-58ae-11ea-8761-7da73daff0fc.png)

This first benchmarks indicates that somewhat better results are obtained using an upper bound of about 15nm. The mean indicates that 20nm is optimal. However, I know that the mean value is usually not a good estimator for this benchmarks. 

Another 2D benchmark coupling lower and upper bounds for this parameter will ensue soon.
